config/initializers/ckeditor.rb

```diff
@@ -9,6 +9,6 @@
 
   config.assets_languages = Rails.application.config.i18n.available_locales.map { |l| l.to_s.downcase }
   config.assets_plugins = %w[balloonpanel balloontoolbar copyformatting image
-                             link magicline scayt table tableselection wsc]
+                             link magicline scayt table tableselection wsc pastefromword]
```

Comment:
Since right now plugins are ordered alphabetically, how about putting it after `magicline`?

Reply:
Not realized about alphabetical sorting 😅